sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/util/SearchPagedIterable.java

```diff
@@ -6,19 +6,61 @@
 import com.azure.core.http.rest.PagedIterableBase;
 import com.azure.core.util.paging.ContinuablePagedIterable;
 import com.azure.search.documents.implementation.models.SearchRequest;
+import com.azure.search.documents.models.FacetResult;
 import com.azure.search.documents.models.SearchResult;
 
+import java.util.List;
+import java.util.Map;
+
 /**
  * Implementation of {@link ContinuablePagedIterable} where the continuation token type is {@link SearchRequest}, the
  * element type is {@link SearchResult}, and the page type is {@link SearchPagedResponse}.
  */
 public final class SearchPagedIterable extends PagedIterableBase<SearchResult, SearchPagedResponse> {
+    private final SearchPagedFlux pagedFlux;
+
     /**
      * Creates an instance of {@link SearchPagedIterable}.
      *
      * @param pagedFlux The {@link SearchPagedFlux} that will be consumed as an iterable.
      */
     public SearchPagedIterable(SearchPagedFlux pagedFlux) {
         super(pagedFlux);
+        this.pagedFlux = pagedFlux;
+    }
+
+    /**
+     * The percentage of the index covered in the search request.
+     * <p>
```

Comment:
This is not valid HTML - you should have the opening `<p>` tag on the same line as the paragraph, and a closing `</p>` tag at the end of the paragraph.